### PR TITLE
[transmission] add chart

### PR DESCRIPTION
**Pull request by ishioni**
**Description of the change**
Adds transmission chart

**Benefits**
This is one of the more popular torrent clients

**Possible drawbacks**

**Applicable issues**

- fixes #

**Additional information**

**Checklist** 
- [X] Chart version bumped in `Chart.yaml` according to [semver](http://semver.org/).
- [X] Title of the PR starts with chart name (e.g. `[home-assistant]`)
- [X] (optional) Chart is using our [common](https://github.com/k8s-at-home/charts/tree/master/charts/common) library as a dependency.
- [ ] (optional) Variables are documented in the README.md

### CODE REVIEW

**Comment by onedr0p** [comment]
Love the config as code, good job!

File: charts/transmission/values.yaml [hunk 1]
```diff
@@ -0,0 +1,184 @@
...
+#     configMap:
+#       # Replace RELEASE-NAME with your actual name
+#       name: RELEASE-NAME-transmission-config
+#
+# additionalVolumeMounts:
+#   - mountPath: /config/settings.json
+#     name: transmission-config
+#     subPath: settings.json
+
+customconfig:
```
Comment (onedr0p) [inline charts/transmission/values.yaml:114]: What about `customConfig`?

File: charts/transmission/templates/configmap.yaml [hunk 2]
```diff
@@ -0,0 +1,11 @@
+{{- if .Values.customconfig.enabled }}
+apiVersion: v1
+kind: ConfigMap
+metadata:
+  name: {{ template "common.names.fullname" . }}-config
+  labels:
+    {{- include "common.labels" . | nindent 4 }}
+data:
+  settings.json: |
+    {{- .Values.customconfig.config | mustToPrettyJson | nindent 4 }}
```
Comment (onedr0p) [inline charts/transmission/templates/configmap.yaml:10]: Same as the other, what about `customConfig`?

**Review by onedr0p — CHANGES_REQUESTED** [review]

File: charts/transmission/values.yaml (hunk 1 above)
Comment (ishioni) [reply charts/transmission/values.yaml]: Done

**Review by onedr0p — APPROVED** [review]

**Pull request merged**